.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - "main"
+      - "*-workflow"
+
+jobs:
+  build:
+    runs-on: ubuntu
```

Comment:
I think you need to replace `ubuntu` to `ubuntu-latest`. This workflow file has been queuing for 8 hours might because of this as the GitHub runner could identifier `ubuntu` image.